app/shared/availability-view/TimelineGroups/TimelineGroup/AvailabilityTimeline/AvailabilityTimelineContainer.spec.js

```diff
@@ -55,27 +52,7 @@ function getState() {
   };
 }
 
-function getWrapper(props) {
-  const defaults = {
-    date: '2016-01-01T00:00:00',
-    id: 'resource-1',
-    onReservationClick: () => null,
-    store: { getState },
-  };
-  return shallow(<AvailabilityTimelineContainer {...defaults} {...props} />);
-}
-
 describe('shared/availability-view/AvailabilityTimelineContainer', () => {
```

Comment:
Those test was removed because it's testing a connected component. Enzyme give warning for giving store as props, those test is not valid also.